data-node/src/main/java/org/graylog/datanode/management/OpensearchProcess.java

```diff
@@ -29,6 +29,7 @@ public interface OpensearchProcess extends ManagableProcess {
 
     Object nodeName();
 
+    boolean isLeaderNode();
```

Comment:
It is really minor, but OpensearchProcessImpl class now misses @Override over "isLeaderNode" method signature.

Reply:
fixed, thx!